pyproject.toml

```diff
@@ -2,12 +2,11 @@
 name = "nautobot-ssot"
 version = "2.0.0"
 description = "Nautobot Single Source of Truth"
-authors = ["Network to Code, LLC <info@networktocode.com>"]
+authors = ["Network to Code, LLC <opensource@networktocode.com>"]
 license = "Apache-2.0"
 readme = "README.md"
-homepage = "https://github.com/nautobot/nautobot-plugin-ssot"
-repository = "https://github.com/nautobot/nautobot-plugin-ssot"
-documentation = "https://nautobot-plugin-ssot.readthedocs.io"
```

Comment:
Is this not used?

Reply:
Not even the right URL, it does 302 to the correct one but not needed.